addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -691,6 +694,8 @@ var FieldDateRange = InputField.extend({
 
         this.$el.daterangepicker(this.dateRangePickerOptions);
         this.$el.on('apply.daterangepicker', this._applyChanges.bind(this));
+        this.$el.on('show.daterangepicker', this._onDateRangePickerShow.bind(this));
+        this.$el.on('hide.daterangepicker', this._onDateRangePickerHide.bind(this));
```

Comment:
it would be better to have them in a new events section at the top:

```js
    events: _.extend({}, InputField.prototype.events, {
        'click': '_onClick',
    }),
```